math/average.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <assert.h>
+
+//average of an array
+
+double average(int avg[]) {
```

Comment:
why avg? is it the array of avgs? Maybe values? What do you think?

Reply:
can you explain this please